### DISCUSSION TITLE
Inery Testnet

### DISCUSSION THREAD

**Discussion by zulna10** [nodesxploit/testnet#7] [Q&A] [answered]
halo bang, tolong tambahin guide untuk inery testent ya

**Comment by nodexone** [comment, marked as answer]
hi bang, oke siapp nanti ane tambahin inery ya, jgn lupa follow dan kasih star ya wkwk

**Reply by zulna10** [reply]
siap bang makasih yaa

**Reply by nodexone** [reply]
siap bang sama sama
